kolibri/plugins/style_guide/assets/src/views/content/components/radio-buttons.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+
+  <component-style-guide
+    class="radio-style-guide"
+    :codeExamplesTemplate="codeExamplesTemplate"
+    :api="api"
+    :requirePath="requirePath"
+  >
+
+    <h1 slot="title">Radio Buttons</h1>
+    <p slot="summary">
+      Radio buttons allow the selection of a single option from a set of 3 or
+      more options.
+    </p>
+
+    <div slot="guidelines-and-usage">
+
+      <h3>Usage</h3>
+      <p>
+        In particular, radio buttons aid the user when they need to see all
+        available options side-by-side.
+      </p>
+      <p>
+        If it is not necessary to see all available options, a dropdown may be
+        used.
+      </p>
+      <p>
+        Selected state of controls generally denote "positives"
+        (yes/selected/etc.)
+      </p>
+
+      <h3>Do not</h3>
+      <ul>
+        <li>
+          Use radio buttons as a filter in a table or a list.
+        </li>
+        <li>
+          Nest radio buttons.
+        </li>
+      </ul>
+
+      <h3>Style</h3>
+      <p>
+        Kolibri utilizes the default styles + padding from Keen UI.
+      </p>
+      <p>
+        Selected option is the action color, other available options are dark
+        grey.
+      </p>
+      <p>
+        Where possible, avoid using icons with radio buttons. If necessary, the
+        icon should sit within the label area, pushing the label to the right.
+      </p>
+
+      <h3>Language</h3>
```

Comment:
Maybe can remove the label "Language" and just have "Labels & option text" as a title